src/styles/blocks/index.scss

```diff
@@ -0,0 +1,22 @@
+html {
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
I moved it to body/card/card__product/card__product-img. and test didn't pass